sklearn/neighbors/unsupervised.py

```diff
@@ -93,7 +93,7 @@ class NearestNeighbors(NeighborsBase, KNeighborsMixin,
       ... #doctest: +ELLIPSIS
       array([[2, 0]]...)
 
-      >>> rng = neigh.radius_neighbors([0, 0, 1.3], 0.4, return_distance=False)
+      >>> rng = neigh.radius_neighbors([[0, 0, 1.3]], 0.4, return_distance=False)
```

Comment:
Not related to this PR but I find the variable name confusing. RNG usually stands for Random Number Generator. I think `rnb` or just `neighbors` would be a better variable name here.